### ISSUE TITLE
[CI] IndexFollowingIT.testIndexFallBehind some times failes on 6.7

### ISSUE THREAD

**Issue by dimitris-athanasiou** [elastic/elasticsearch#38617]
Build failure: https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.7+internalClusterTest/137/console

Reproduce with: 

```
./gradlew :x-pack:plugin:ccr:internalClusterTest -Dtests.seed=8A5DBECB542A869B -Dtests.class=org.elasticsearch.xpack.ccr.IndexFollowingIT -Dtests.method="testIndexFallBehind" -Dtests.security.manager=true -Dtests.locale=es-CU -Dtests.timezone=America/Vancouver -Dcompiler.java=11 -Druntime.java=8
```

Failure:

```
ERROR   0.59s J7 | IndexFollowingIT.testIndexFallBehind <<< FAILURES!
14:04:13    > Throwable #1: org.apache.lucene.store.AlreadyClosedException: engine is closed
14:04:13    > 	at __randomizedtesting.SeedInfo.seed([8A5DBECB542A869B:6DB7F91E3A82A6DD]:0)
14:04:13    > 	at org.elasticsearch.index.shard.IndexShard.getEngine(IndexShard.java:2264)
14:04:13    > 	at org.elasticsearch.index.shard.IndexShardTestCase.getDocIdAndSeqNos(IndexShardTestCase.java:698)
14:04:13    > 	at org.elasticsearch.xpack.CcrIntegTestCase.getDocIdAndSeqNos(CcrIntegTestCase.java:490)
14:04:13    > 	at org.elasticsearch.xpack.CcrIntegTestCase.lambda$assertIndexFullyReplicatedToFollower$4(CcrIntegTestCase.java:454)
14:04:13    > 	at org.elasticsearch.test.ESTestCase.assertBusy(ESTestCase.java:836)
14:04:13    > 	at org.elasticsearch.xpack.CcrIntegTestCase.assertIndexFullyReplicatedToFollower(CcrIntegTestCase.java:453)
14:04:13    > 	at org.elasticsearch.xpack.ccr.IndexFollowingIT.testIndexFallBehind(IndexFollowingIT.java:997)
14:04:13    > 	at java.lang.Thread.run(Thread.java:748)
```

I managed to reproduce this locally by running:

```
./gradlew :x-pack:plugin:ccr:internalClusterTest -Dtests.class=org.elasticsearch.xpack.ccr.IndexFollowingIT -Dtests.method="testIndexFallBehind" -Dtests.iters=20
```

It failed once in my first try.

**Comment by cbuescher** [comment]
Another one on 7.0
https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+7.0+matrix-java-periodic/ES_BUILD_JAVA=openjdk12,ES_RUNTIME_JAVA=java8,nodes=immutable&&linux&&docker/4/console

**Comment by cbuescher** [comment]
Also muted this on 7.0 with b45923cb51a8e2517d625d01d184934962a93f04

**Comment by jasontedor** [comment]
@martijnvg Could you remove the awaits fix in 6.7 and 7.0 for this test per #38630?

**Comment by martijnvg** [comment]
@jasontedor whoops, I will unmute this test now.
